setup.cfg

```diff
@@ -70,8 +70,10 @@ doc_files = README.txt
 [tool:pytest]
 testpaths = pydoctor/test
 python_files = test_*.py
-addopts = --doctest-glob='*.doctest'
+addopts = --doctest-glob='*.doctest' --strict-config --strict-markers
 doctest_optionflags = ELLIPSIS IGNORE_EXCEPTION_DETAIL
+xfail_strict = true
+filterwarnings = error
```

Comment:
```suggestion
filterwarnings =
    error
    ignore:Writer 'InMemoryWriter' does not support HTML template customization with --template-dir\.:UserWarning:pydoctor.driver
    ignore:Using or importing the ABCs from 'collections' instead of from 'collections\.abc' is deprecated since Python 3\.3, and in 3\.10 it will stop working:DeprecationWarning: pydoctor.templatewriter.util
```